tests/parametric/test_headers_tracecontext.py

```diff
@@ -786,27 +786,26 @@ def test_tracestate_w3c_p_extract(self, test_agent, test_library):
 
     @missing_feature(context.library < "python@2.7.0", reason="Not implemented")
     @missing_feature(context.library == "dotnet", reason="Not implemented")
-    @missing_feature(context.library == "php", reason="Not implemented")
-    @missing_feature(context.library == "nodejs", reason="Not implemented")
+    @missing_feature(context.library < "php@0.99.0", reason="Not implemented")
+    @missing_feature(context.library < "nodejs@5.6.0", reason="Not implemented")
     @missing_feature(context.library == "java", reason="Not implemented")
     @missing_feature(context.library == "cpp", reason="Not implemented")
-    @missing_feature(context.library == "ruby", reason="Not implemented")
+    @missing_feature(context.library < "ruby@2.0.0", reason="Not implemented")
     @missing_feature(context.library == "golang", reason="Not implemented")
     def test_tracestate_w3c_p_inject(self, test_agent, test_library):
         """
         Ensure the last parent id is propagated according to the W3C spec
         """
         with test_library:
             with test_library.start_span(name="new_span") as span:
-                pass
-
-            headers = test_library.inject_headers(span.span_id)
+                headers = test_library.inject_headers(span.span_id)
```

Comment:
The node tracer deletes a span from the parametric app after the span is finished. To run the tests with the node app we need to get the headers before the span is finished.